ci-operator/step-registry/osd-ccs/cluster/provision/gcp/osd-ccs-cluster-provision-gcp-commands.sh

```diff
@@ -4,6 +4,22 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+# Select the OCP version according to the env vars OPENSHIFT_VERSION and CHANNEL_GROUP
+function ocp_version_selector() {
+  versionList=$(mktemp)
```

Comment:
Actually I did not find out the difference with the origin one (the only difference is saving the output into a file)

Reply:
Right, I'd like to save the versions into a file (one line one version), and then cat it for grep & tail.

Reply:
Another diff is I added "^" when grep for "${OPENSHIFT_VERSION}", see line 15. Thanks!

Reply:
I think you could update the old line to add `^`. I am confusing with Jianlin too, there is useless to store the versionlist in a file.